backend/common/src/main/java/ai/verta/modeldb/common/artifactStore/storageservice/s3/RefreshS3ClientCron.java

```diff
@@ -41,7 +42,7 @@ public void run() {
       createAndRefreshNewClient(awsCredentials);
     } catch (Throwable ex) {
       LOGGER.error("Failed to refresh S3 Client: " + ex.getMessage(), ex);
-      throw ex;
+      throw new RuntimeException(ex);
```

Comment:
Is there really any value in throwing here? We could just return and be done. Throwing won't end up generating any actual information, since the exception will be swallowed by the executor.